### ISSUE TITLE
flexbox footer not working on IE11

### ISSUE THREAD

**Issue by hisakonog** [hisakonog/ilang-book-website-2.0#1]

**Comment by hisakonog** [comment]
> [Since](https://github.com/philipwalton/flexbugs#3-min-height-on-a-flex-container-wont-apply-to-its-flex-items) the body element will never have any content below it, and since having a vertical scroll bar appear when there's a lot of content on the page is usually the desired behavior, substituting `height` for `min-height` will almost always work

Flexbox footer doesn't work on Safari when `min-height` is replaced by `height`, footer floats in the middle of the page. 
Adding another layer of div (flex-container) inside body and move min-height to flex-container  like below solved the floating problem on Safari, but footer slides up when the page content is sparse (on Chrome, Firefox AND Safari). 

```
<body>
<div class="flex-container"> 
...
</div> 
</body>
```

```
body{
    display: -webkit-box;
    display: -moz-box;
    display: -ms-flexbox;
    display: -webkit-flex;
    display: -moz-flex;
    display: -ms-flex;
    display: flex; 
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    -mox-flex-direction: column;
    flex-direction: column;
    /* min-height: 100%; */
    padding-top: 165px;
}

.flex-container{
    min-height: 100%;
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    -mox-flex-direction: column;
    flex-direction: column;
}
```

**Comment by hisakonog** [comment]
Try IE [specific hack](https://philipnewcomer.net/2014/04/target-internet-explorer-10-11-css/) to insert height in css. 
Update: it doesn't fix. When height overrides min-height the footer sticks at the bottom of the content, and if the content is sparse the footer floats in the middle of the screen.

**Comment by hisakonog** [comment]
Uses javascript to [push the footer down](http://stackoverflow.com/questions/16260498/push-footer-to-bottom-when-page-is-not-full). Not as nice as flexbox but at least the site works the same on IE...

**Comment by hisakonog** [comment]
Closed via 8e1bc7b
